src/components/JiraResults.js

```diff
@@ -75,9 +75,9 @@ const JiraResults = (props) => {
 							{props.jiras.map((jira) => getResultLine(jira))}
 						</Alert>
 						<br />
-						<p className="btn btn-primary" onClick={copyToClipboard}>
+						<Button className="btn btn-primary" onClick={copyToClipboard} disabled={props.jiras[0].summary === ""}>
```

Comment:
This condition will not work expected in these cases:
## Test Case
- The user has 2 rows
- The summary of the first row is empty
- The summary of the 2nd row is NOT empty
### Actual outcome
The "Copy Results" button is disabled
### Expected outcome
The "Copy Results" button should be enabled

Reply:
okie i will work on that @anushibin007

Reply:
functionality done @anushibin007 ✌️